controls/treeview/checkboxes.md

```diff
@@ -18,6 +18,13 @@ You can control the state propagation by setting the `CheckBoxMode` property (en
 * `Independent`&mdash;Specifies that the checkboxes are checked and unchecked independently. The checked state of the parent item is not propagated to its children.
 * `Recursive`&mdash;Specifies that the checkboxes are checked and unchecked recursively. The checked state of the parent item is propagated to its children.
 
+## Recursive CheckBox Mode and Load Children on Demand
+
+Consider the following scenarios when using the TreeView `Recursive` `CheckBoxMode` in a combination with the `LoadChildrenOnDemand` feature:
+
+1. When checking the checkbox element of the parent item without loading the children, this parent item is added to the `CheckedItems` collection (the children are not).
```

Comment:
If these aren't steps that must be performed one after the other, consider using an ordered list.